CSS/about-myself.css

```diff
@@ -0,0 +1,137 @@
+.about-myself {
+  background-color: #fff;
+  width: 100%;
+  height: 1200px;
+  padding: 0 24px;
+  border-top-right-radius: 100px;
+  margin-top: 60px;
+}
```

Comment:
- [ ] Kindly  make sure that the about section takes the max width  so that it will remove the scrolling button and should have the same size as your home page and is better to use margin to add the space  instead of using padding which adds the space inside your contena :grimacing: 

![image](https://user-images.githubusercontent.com/95297251/185510158-31a6256a-d289-4f60-b173-6f74df8de768.png)

Reply:
can you explain more please, I am very sorry I don't know what do you mean, if you could highlight it if possible.